templates/kapeta/block-type-cli/rollup.config.mjs

```diff
@@ -0,0 +1,42 @@
+import packageJson from './package.json' assert { type: 'json' };
```

Comment:
Isn't the template destination comment missing here?

Reply:
3293df2fb554e8af91e3acd7ae68bb8838bf4e73